sources/ElkArte/Modules/Random/Display.php

```diff
@@ -74,6 +76,23 @@ public static function followup_message_list($messages)
 		$context['follow_ups'] = followupTopics($messages, self::$includeUnapproved);
 	}
 
+	/**
+	 * Show likes.
+	 *
+	 * @param int[] $messages
+	 */
+	public static function load_likes($messages)
+	{
+		global $modSettings, $context, $txt;
+
+		if (!empty($modSettings['likes_enabled']) && !empty($messages))
+		{
+			require_once(SUBSDIR . '/Likes.subs.php');
+			$context['likes'] = loadLikes($messages, true);
+			theme()->getLayers()->addBefore('load_likes_button', 'body');
```

Comment:
Which this being loaded, there should be no need to call the template_load_likes_button_above() from display template init ?